mozilla/foundation.mozilla.org
# Redirect middleware for selecting a redirect page

Closes #6114 

To test this:
 * Setup a new locale in the /cms/ admin (ie. french)
 * Create a redirect from /test-page to a page you've chosen (ie chose the english blog page)
 * Change your browser settings to another language (ie. french)
 * Go to http://localhost:8000/test-page 
 * It _should_ take you to http://localhost:8000/fr/blog/ even though you selected the english blog page for the redirect

Also worth testing:
* Set your browser language to an unsupported language from the site (ie. If there are no Locales for spanish, set your browser to spanish) 
* Test the same redirect — because the Spanish Locale doesn't exist in the cms, it should redirect to the main selected page. 

**Note** This PR is _just_ for selecting a page when creating a new redirect. And this only works if the users language code is set in the request.

**Link to sample test page**: https://foundation-s-6114-ensur-kcjryj.herokuapp.com/english-selected-but-goes-to-es
